.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,11 +6,10 @@ to the issue.
 
 # Checklist
 
-- [ ] Commit sequence broadly makes sense
-- [ ] Commits have useful messages
+- [ ] Commit sequence broadly makes sense and commits have useful messages
 - [ ] New tests are added if needed and existing tests are updated
-- [ ] Any changes are noted in the [changelog](https://github.com/input-output-hk/cardano-ledger/blob/master/CHANGELOG.md)
-- [ ] Code is formatted with [`fourmolu`](https://github.com/fourmolu/fourmolu) (use `scripts/fourmolize.sh`)
-- [ ] Cabal files are formatted (use `scripts/cabal-format.sh`)
-- [ ] `hie.yaml` has been updated (use `scripts/gen-hie.sh`)
+- [ ] Any changes are noted in the `CHANGELOG.md` for affected package
+- [x] Code is formatted with [`fourmolu`](https://github.com/fourmolu/fourmolu) (use `scripts/fourmolize.sh`)
+- [x] Cabal files are formatted (use `scripts/cabal-format.sh`)
+- [x] [`hie.yaml`](https://github.com/input-output-hk/cardano-ledger/blob/master/hie.yaml) has been updated (use `scripts/gen-hie.sh`)
```

Comment:
Yeah, because they are enforced by CI. So, there is no reason to manually check it. All three are left mostly as a reminder and as instructions on how to do this formatting for new contributors.